docs/versions.rst

```diff
@@ -18,14 +18,14 @@ TEAL Version PyTeal Version
 6            >= 0.10.0
```

Comment:
@jdtzmn 2 requests here relating to our desire to have the change take effect as of AVM 7:
* Can you extend the table to include AVM 7 with expected version?  As of writing, I think it'll be v0.14.0.
* Can you base the PR off of `teal7` (https://github.com/algorand/pyteal/pull/416)?
  * I realize the request creates conflicts.  I apologize for not calling it out earlier.